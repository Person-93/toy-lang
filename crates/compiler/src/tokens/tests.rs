use super::*;

use insta::{assert_ron_snapshot, glob};
use logos::Logos;
use std::fs;

#[test]
fn snapshot() {
  glob!("toys/*.toy", |file| {
    let text = fs::read_to_string(file).unwrap();
    assert_ron_snapshot!((Token::lexer(&text).spanned().collect::<Vec<_>>()));
  })
}
