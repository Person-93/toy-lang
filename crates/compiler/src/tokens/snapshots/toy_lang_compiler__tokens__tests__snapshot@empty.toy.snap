---
source: crates/compiler/src/tokens/tests.rs
assertion_line: 11
expression: "(Token::lexer(&text).spanned().collect::<Vec<_>>())"
input_file: toys/empty.toy

---
[]
